javaagent-spi/src/main/java/io/opentelemetry/javaagent/spi/exporter/MetricServer.java

```diff
@@ -23,4 +24,11 @@
    * @param config The configuration
    */
   void start(MetricProducer producer, Properties config);
+
+  /**
+   * Returns names of metric servers supported by this factory.
```

Comment:
i think it's a little confusing why we support multiple names without a bit of explanation:

```suggestion
   * Returns names of metric servers supported by this factory.
   *
   * <p>Multiple names are useful for enabling a pair of span and metric exporters using the same
   * name, while still having separate names for enabling them individually.
```